src/hooks/core/useCoreChatSystemHook.ts

```diff
@@ -33,7 +32,7 @@ import {getChannelListInfo} from '../../utils/string/StringUtils';
 
 const useCoreChatSystemHook = () => {
   const {localDb, refresh} = useLocalDatabaseHook() as UseLocalDatabaseHook;
-  const {anonProfileId, signedProfileId, profile} = useProfileHook();
+  const {anonProfileId, signedProfileId} = useProfileHook();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `profile` object was removed from the destructuring of `useProfileHook()`. If this object is not used elsewhere in the code, this change is fine. However, if it's used later in the code, this could cause a reference error. Please ensure that the `profile` object is not required anywhere else in this file.
```diff
-  const {anonProfileId, signedProfileId, profile} = useProfileHook();
+  const {anonProfileId, signedProfileId} = useProfileHook();
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->